go.mod

```diff
@@ -3,12 +3,14 @@ module github.com/elastic/terraform-provider-ec
 go 1.19
 
 require (
+	github.com/blang/semver v3.5.1+incompatible
 	github.com/blang/semver/v4 v4.0.0
 	github.com/elastic/cloud-sdk-go v1.10.0
 	github.com/go-openapi/runtime v0.24.1
 	github.com/go-openapi/strfmt v0.21.3
 	github.com/hashicorp/terraform-plugin-framework v0.11.1
```

Comment:
It looks like there is already v.0.12.0. Does it make sense to update it?

Reply:
FYI: I started working on this. Currently ironing out an issue with ProviderData and unit tests/mocking. Will open a new PR with the changes once this is done..